src/host/rpcconnections.h

```diff
@@ -172,6 +171,16 @@ namespace asynchost
       return s->second->write(len, data);
     }
 
+    bool stop(int64_t id)
+    {
+      // Invalidating the TCP socket will result in the handle being close. No
```

Comment:
Typo: Should be "being close**d**".

And "read from" isn't necessarily true - between this point and the enclave processing the `tls_close` message, additional messages about this `id` may be written by the enclave and read by the host.

Reply:
So we need a null-check in `RPCConnections::write`, as `s->second` may be `nullptr`.